### DISCUSSION TITLE
Gemini's response cut off in half way

### DISCUSSION THREAD

**Discussion by TheValkyrja** [danny-avila/LibreChat#4755] [Troubleshooting]
### What happened?

When using Gemini endpoint and setting the model as gemini-exp-1114, AI's response cut off halfway, showing "The response is incomplete; it's either still processing, was cancelled, or censored. Refresh or try a different prompt." after refresh, the response shown as "network error". This only happens while using gemini-exp-1114 the specific model but not happening while using other geminis.

i found similar problem in #4440 , however didn't experienced the problem in gemini-1.5-pro-002 but in gemini-exp-1114.

### Steps to Reproduce

1.set the model to gemini-exp-1114
2.give a relatively long promt
3.the response cut in half way.

### Screenshots

![image](https://github.com/user-attachments/assets/cb518560-fb3e-482a-9081-530c8422cd18)
![image](https://github.com/user-attachments/assets/eac7532c-e78a-4f58-9f52-7bbc4389ade4)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Try updating, I don't have this issue with `gemini-exp-1114`

**Reply by svnkxmza** [reply]
I have the same problem. Long answers are interrupted. Short answers aren't.
![2024-11-20 032800](https://github.com/user-attachments/assets/ee06f498-e902-4cda-801e-f4221c6e48dc)

**Reply by TheValkyrja** [reply]
> Try updating, I don't have this issue with `gemini-exp-1114`

i have. this is the latest version, and i also tried re-install docker and rebuild the image, didn't work.

**Reply by kiyo-e** [reply]
Same here.

**Reply by insilications** [reply]
I have this issue with gemini-exp-1121. It is fully updated to the last docker image

**Comment by danny-avila** [comment]
At the time of moving this to discussions, it wouldn't happen to me. Now that I'm trying again, I'm also experiencing the behavior. I'm taking a closer look at this today.

**Comment by danny-avila** [comment]
Should be fixed here: https://github.com/danny-avila/LibreChat/pull/4781
